eng/Versions.props

```diff
@@ -53,6 +53,11 @@
     <MicrosoftBuildLocatorVersion>1.2.6</MicrosoftBuildLocatorVersion>
     <MicrosoftBuildRuntimeVersion>15.3.409</MicrosoftBuildRuntimeVersion>
     <MicrosoftBuildTasksCoreVersion>15.3.409</MicrosoftBuildTasksCoreVersion>
+    
+    <NuGetVisualStudioContractsVersion>5.7.0</NuGetVisualStudioContractsVersion>
```

Comment:
Why is this not 5.8.0? Visual Studio is using that version no?

Reply:
NuGet won't ship a 5.8.0 package until VS 16.8 ships GA. Hence that package version doesn't exist yet.

Reply:
I don't know.  @nkolev92 @jasonmalinowski ?

Reply:
This _shouldn't_ matter as [this](https://devdiv.visualstudio.com/DevDiv/_git/VS?path=%2Fsrc%2FProductData%2FAssemblyVersions.tt&version=GBmain&line=91&lineEnd=92&lineStartColumn=1&lineEndColumn=1&lineStyle=plain) will only cause 5.8.0 to load (as long as this is only in-process). @zivkan are we certain that all the work here is only happening in-process?

Reply:
This property is used in the `PackageReference`, so it needs to be a package version that exists, otherwise your restore will fail: https://github.com/dotnet/roslyn/pull/46022/files#diff-891cc3ee7e8cadb5b5e77195e9c826a7R175

You raised the in-process question in this other comment thread, so let's discuss that there: https://github.com/dotnet/roslyn/pull/46022#discussion_r480352850